pages/sync/generic-storage.en.mdx

```diff
@@ -0,0 +1,67 @@
+# Generic Versioned Storage Sync
```

Comment:
can you also make sure to add the .pt-br version of this? can stay english, but at least those users won't get a 404 then (plus translators will be able to see which files are not yet translated and possibly offer a PR

Reply:
Will do !